### PR TITLE
Fk 3252 comments responsive

### PR DESCRIPTION
**Pull request by andreiradoi**

### CODE REVIEW

File: server/api/notes_service.go [hunk 1]
```diff
@@ -290,6 +291,28 @@ func (s *NotesService) UploadMedia(ctx context.Context, payload *notes.UploadMed
 	}, nil
 }
 
+func (s *NotesService) DeleteMedia(ctx context.Context, payload *notes.DeleteMediaPayload) error {
```
Comment (jlewallen) [inline server/api/notes_service.go:294]: This appears to also need a permissions check to make sure users can only manipulate media on stations they own.

File: server/api/station_service.go [hunk 2]
```diff
@@ -209,6 +209,36 @@ func (c *StationService) Transfer(ctx context.Context, payload *station.Transfer
...
+        }
+        return err
+    }
+
+    p, err := NewPermissions(ctx, c.options).ForStation(updating)
+    if err != nil {
+        return err
+    }
+
+    if err := p.RequireAdmin(); err != nil {
```
Comment (jlewallen) [inline server/api/station_service.go:228]: If my memory serves RequireAdmin checks for site admins, so it's pretty restrictive. We probably want to do the same check we need above, for this. Simply changing to CanModify may be enough.

**Comment by jlewallen** [comment]
I would also love to start seeing automated tests for functionality. There is more precedent for the backend tests right now (in station_service_test.go, for example) so it should be fairly easy to get one of those in for the new endpoint methods.

**Comment by andreiradoi** [comment]
@jlewallen I've made some small css changes for post button positioning and textarea maxheight. 
Achil made the permissions changes.
If everything looks good could we merge this PR for now and deal with the tests later on?

**Pull request merged**